charts/datadog/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v1
 name: datadog
-version: 3.83.0
+version: 3.83.1
```

Comment:
do you mind running `.github/helm-docs.sh` to update the version in the `README.md`?